Description: Fix webob requirements.txt
 This avoid loads of unit tests errors.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-02-03

--- cinder-2013.2.1.orig/requirements.txt
+++ cinder-2013.2.1/requirements.txt
@@ -24,5 +24,5 @@ SQLAlchemy>=0.7.8
 sqlalchemy-migrate>=0.7.2
 stevedore>=0.10
 suds>=0.4
-WebOb>=1.2.3,<1.3
+WebOb>=1.2.3
 wsgiref>=0.1.2
